### DISCUSSION TITLE
Missing options in the PELE parameters options

### DISCUSSION THREAD

**Discussion by SergiR1996** [NBDsoftware/nbdsuite_issues#7] [Q&A]
I would like to add the "**linkstoOmit**" from the "**ANM**" block and the "**superpositionSelection**" from the "**RMSD**" metric in the "**Metrics**" block. Both will have a similar structure as the one from "selection_group_1" of the "com_distance" metric type. Many other options from the PELE control file should be also modifiable, this is just to name a few. Another option would be to allow the user to add a control file of its own (or a part of it, meaning a block, like the "ANM" block, for instance).

**Comment by marcuspfalls** [comment]
Hi Sergi! I'm going to work on this now. I'll add in the "linksToOmit" and "superpositionSelection" as you requested. If you have other parameters you want me to add, can you send them? Thanks!
With regards to users submitting their own control file (or sections of one), I'll chat to Martí about it.

**Reply by marcuspfalls** [reply]
By the way, can you send me some conf files that use these parameters as examples? Thanks

**Reply by SergiR1996** [reply]
Sure, I attach them in this reply (the PELE conf files).

[control_files_with_specified_flags.zip](https://github.com/NBDsoftware/nbdsuite_issues/files/13704922/control_files_with_specified_flags.zip)

**Comment by marcuspfalls** [comment]
Hi Sergi, I've looked at the files and the code. To answer your initial question: 

linksToOmit for both chains and links is already implemented. To omit chains like in the example you sent me, you need to use the parameter `anm_chains_to_omit` with the chains in a list. To omit links, use `anm_links_to_omit`. I'm now aware that the documentation is unclear/erronous on this so I'll edit that to clarify. 

I'll add in the superpositionSelection parameter too. 

You mentioned other options that you also want to be able to modify, could you let me know what they are and I'll add them in this branch too along with the superPosition changes? Cheers!

Finally, I've also discussed with Martí about the option to add a PELE control file as a parameter. We agree it's a good idea for more advanced users so we've added it to our sprint board to implement in the next few weeks alongside a tutorial.

**Comment by SergiR1996** [comment]
Hi Marcus!

Okay, that's great! Martí told me that he would add them, but I did not know that they were already available. I will think of other parameters, but I think the main ones I use are covered. 

Yes, adding the control file is definitively a good option, since a lot of times we reuse the same control file from another project, but we just basically change the input PDB files, and thus, the list of constraints. Actually, it would be a good idea that you can pass him a "templatized" control file where he uses the values of the defined parameters and sets up the rest from the default of the Nostrum suite.

**Comment by marcuspfalls** [comment]
Hi Sergi, I've been able to replicate the conf files inside /shared/work/sroda/DD/Navigo2/PELE/32CPU/SpA2_32CPU/ using the code changes that I've implemented. We'll merge this into the main code on Friday. In the meantime let me know if there's any other additions that you want added to the conf file, otherwise we'll close this issue. Cheers!
